include/zcbor_common.h

```diff
@@ -74,6 +74,11 @@ do { \
 #define ARRAY_SIZE(arr) (sizeof(arr) / sizeof((arr)[0]))
 #endif
 
+#if SIZE_MAX == UINT32_MAX || SIZE_MAX == UINT64_MAX
+#define ZCBOR_SUPPORTS_SIZE
```

Comment:
This should have a bit of documentation.

Reply:
Added comment.